.mise.toml

```diff
@@ -0,0 +1,16 @@
+[tasks.assets]
+description = "Create assets with vhs"
+run = """
+#!/usr/bin/env bash
+# Create VHS recordings of all tape files in the assets directory
+for i in $(ls -1 assets/*.tape); do
+    vhs $i
```

Comment:
this is cool!

Reply:
unfortunately `vhs` has dependencies on `ttyd` and `ffmpeg` which probably cannot be easily installed/defined via mise tools options